global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "sdk": {
-        "version": "8.0.0",
+        "version": "9.0.0",
```

Comment:
Slightly OT: After updating MSVS to v17.12.1, I had to generally remove the patch version in order to build anything, i.e. just `8.0`.

Reply:
That worked for me, I removed the artifacts though. (and 17.2 is required for .net9). Is .net9 really used if this is not changed.

Sidenote: I have not seen any problem with this included in my daily build. #11996 hanged very frequently with .net9, but it may hang with .net8 too, no problems after excluding that.
I guess I play around with this some time and we decide later if to migrate or not.

Reply:
That's strange. In a day-to-day work I use the internal previews of VS (17.12, .13, etc.) and work in multiple repos with global.json, and in none of those we had to remove the patch version...

Reply:
The MSVS Installer seems to have wiped all 8.x.y SDKs.
I wonder whether the 9.0.100 is used as fallback (in a compatibility mode?) if just "8.0" is requested.

![image](https://github.com/user-attachments/assets/2f724e50-fdaf-4447-8677-46df66a1ecb6)

Reply:
That's bizarre... Maybe report this at https://github.com/dotnet/sdk?

Reply:
They sell it as a feature: https://github.com/dotnet/sdk/issues/45241#issuecomment-2511746214